### PR TITLE
About page copy changes

### LINKED ISSUE #175: Revise the about page copy
Don't think the information is really accurate anymore. Things to consider:

- Is the story intro still accurate?
- Let's describe the demo state, what's included and what to expect going forward
- Let's expand on "How to contribute". Just opening an issue is fine for small bugs, but not helpful if someone wants to help with design, localization, story, etc
- Update list of contributors
- Include a simple way to give feedback on the learning experience

What else?

I suggest starting a Google doc for drafting up a new and improved version of this page.

### PR DESCRIPTION
**Pull request by afterburn**
This PR closes #175. Noticed some issues with the translations for the `dutch` version, but these should go away after #245.

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
One copy fix I missed in figma. 

Can you change " You open the holocat by booping its nose curious to hear what the it has to say..."

to 

" You open the holocat by booping its nose curious to hear what it has to say..." I've also updated this in figma.

Also my pseudonym + github url is incorrect. It should be spelled ecurrencyhodler not ecurrencyholder. Can you link to twitter instead as well? https://twitter.com/ecurrencyhodler

I checked links to other contributors and they work.

**Comment by afterburn** [comment]
Ah sorry, my bad. I fixed your username + url and updated the copy.

**Comment by ecurrencyhodler** [comment]
tACK! Great work. Let's get 1 code review then merge.

File: app/[lang]/about/page.tsx [hunk 1]
```diff
@@ -3,131 +3,116 @@ import Footer from 'components/Footer'
...
-            <br />
-            The price of bitcoin is falling. There is a knock at your door, and
-            when you open it, a yellowed, age-weathered envelope has been left
-            for you. There’s no name or return address. What could be inside?
+          <p className="mt-4 text-center text-4xl">{t('about.subtitle')}</p>
+          <p className="mt-8 font-nunito text-xl">{t('about.intro')}</p>
+
+          <Divider />
+
+          <h3 className="text-5xl">{t('about.project.title')}</h3>
```
Comment (vinayaksh42) [inline app/[lang]/about/page.tsx:41]: The size of the heading is correct for the desktop view but for mobile view, it needs to be `36px` - `text-4xl` as per the [figma mockup ](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2323-26516&t=eHsCzdoNoh1RCi6l-0)

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
All looks good to me, just a minor change for mobile view

**Comment by ecurrencyhodler** [comment]
For mobile, I think @GBKS specifically wanted "What it's about"
image

**Comment by ecurrencyhodler** [comment]
tACK. @vinayaksh42 can you check the requested change?

**Pull request merged**